cloud/pkg/devicecontroller/controller/downstream_test.go

```diff
@@ -0,0 +1,106 @@
+package controller
```

Comment:
done, I assume one of the test case fails due to some other reasons. [this PR](https://github.com/kubeedge/kubeedge/pull/5926) is also pending since weeks, PTAL : )